Tests/RockLib.Messaging.SQS.Tests/SQSTests.cs

```diff
@@ -412,6 +412,54 @@ public static void SQSReceiverMessageWithUnpackSNSTrueCorrectlyUnpacksSNSMessage
             sqsReceiverMessage.Headers["TopicARN"].Should().Be("arn:PutARealARNHere");
         }
 
+        [Fact]
+        public static void SQSReceiverMessageHasSQSMessageIDSetCorrectly()
+        {
+            var message = new Message
+            {
+                MessageId = (new Guid()).ToString(),
+                Body = @"{""TestMessageItem"":""ThisIsAFakeItem""}"
+            };
+
+            var sqsReceiverMessage = new SQSReceiverMessage(message, c => Task.FromResult(0), c => Task.FromResult(0), unpackSNS: false);
+
+            var messageId = sqsReceiverMessage.Headers.GetValue<string>("SQS.MessageID");
+
+            messageId.Should().NotBeNull();
+
+            sqsReceiverMessage.Dispose();
```

Comment:
Can you update line 424 to `using var sqsReceiverMessage` and then you can remove the `sqsReceiverMessage.Dispose()` on line 430.  That way it's consistent with the other tests in the file.